litmus-portal/frontend/src/views/AnalyticsDashboard/MonitoringDashboardPage/InfoDropdown/index.tsx

```diff
@@ -0,0 +1,204 @@
+  };
+
+  const handleMetricSelect = (selectedMetric: string) => {
+    const newSelectedMetrics = selectedMetrics?.includes(selectedMetric)
+      ? selectedMetrics?.filter((id) => id !== selectedMetric)
+      : [...(selectedMetrics ?? []), selectedMetric];
+    setSelectedMetrics(newSelectedMetrics);
+    postPanelSelectionRoutine(newSelectedMetrics);
+  };
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.header}>
+        <Typography className={classes.headerText}>
+          {t('analyticsDashboard.monitoringDashboardPage.infoDropdown.header')}
+        </Typography>
+        <ButtonFilled
+          className={classes.button}
+          onClick={() => {
+            closeInfo();
+          }}
+        >
+          <Typography className={classes.closeText}>
+            {t('analyticsDashboard.monitoringDashboardPage.infoDropdown.close')}
+          </Typography>
+          <img
+            src="/icons/closeIcon.svg"
+            alt="close"
+            className={classes.closeIcon}
+          />
+        </ButtonFilled>
+      </div>
+      <div className={classes.body}>
+        <div className={classes.infoSectionElement}>
+          <Typography className={classes.sectionHeader}>
+            {t(
+              'analyticsDashboard.monitoringDashboardPage.infoDropdown.subHeading1'
+            )}
+          </Typography>
+          <div className={classes.dashboardMetaDataItem}>
+            <Typography className={classes.infoKey}>
+              {t(
+                'analyticsDashboard.monitoringDashboardPage.infoDropdown.metaData1'
+              )}
+            </Typography>
+            <Typography className={classes.infoValue}>
+              {' '}
+              {dashboardConfigurationDetails.name}{' '}
+            </Typography>
+          </div>
+          <div className={classes.dashboardMetaDataItem}>
+            <Typography className={classes.infoKey}>
+              {t(
+                'analyticsDashboard.monitoringDashboardPage.infoDropdown.metaData2'
+              )}
+            </Typography>
+            <div>
+              <Typography className={classes.infoValue}>
+                <img
+                  src={`/icons/${
+                    dashboardConfigurationDetails.type === 'Kubernetes Platform'
+                      ? 'kubernetes-platform'
+                      : 'sock-shop'
+                  }.svg`}
+                  alt="dashboard Icon"
+                  className={classes.dashboardIcon}
+                />{' '}
+                {dashboardConfigurationDetails.type}{' '}
+              </Typography>
+            </div>
+          </div>
+          <div className={classes.dashboardMetaDataItem}>
+            <Typography className={classes.infoKey}>
+              {t(
+                'analyticsDashboard.monitoringDashboardPage.infoDropdown.metaData3'
+              )}
+            </Typography>
+            <Typography className={classes.infoValue}>
+              {' '}
```

Comment:
please remove {' '} it might have been added by `prettier` on its own

Reply:
yes you are correct @whitetiger1399 will remove this